testing/test-helper.js

```diff
@@ -27,7 +27,7 @@ export function stubService(sandbox, win, serviceId, method) {
     service[method] = stub;
     return service;
   });
-  return sandbox.stub(service, method);
+  return stub;
```

Comment:
Nope, this was a bugfix! Register above will only happen if it's unregistered, but we still want to make sure this method is stubbed when the service is already registered.